components/reviews/ReviewItem.module.css

```diff
@@ -0,0 +1,29 @@
+.review {
+    display: flex;
+    flex-direction: row;
+    height: 6rem;
+    border-bottom: 4px solid #000;
+}
+
+.partial {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-around;
+    align-items: center;
+    background-color: #f0eeed;
+    border-right: 2px solid #d9d6d4;
+    border-radius: 8px;
+    width: 17rem;
+}
+
+.name {
+    margin: auto;
+    font-size: 1.3rem;
+    font-weight: bold;
+}
+
+.comment {
+    overflow-wrap: break-word;
+    overflow-y: scroll;
```

Comment:
I think that this causes the problem

Reply:
If we remove overflow-y, maybe we could replace it with a max number of characters in the review?

Reply:
Yea that sounds like a good idea 👍  and it should work just adding a maxLength on the input